docs/articles_en/documentation/openvino-ir-format/operation-sets/operation-specs/movement/col2im-15.rst

```diff
@@ -0,0 +1,198 @@
+
+**Detailed description**
+
+Consider an ``input`` tensor containing batches of image blocks of shape ``(N, C * Product(kernel_size), L)``, where:
+
+* ``N`` is the batch dimension,
+* ``C * Product(kernel_size)`` is the number of elements within a block (each block contains ``Product(kernel_size)`` vectors containing values from each channel ``C``),
+* ``L`` is the total number of blocks calculated as follows:
+
+.. math::
+
+    L = \prod_{d=1}^{2} \lfloor \frac{({\text{{output\_size}}[d] + \text{{pads\_begin}}[d] + \text{{pads\_end}}[d] - \text{{dilation}}[d] \times (\text{{kernel\_size}}[d] - 1) - 1}}{{\text{{stride}}[d]}}) + 1\rfloor
+
+
+where ``d`` is over all spatial dimensions.
+
+The ``input`` blocks are being moved into the ``output`` tensor of shape ``(N, C, output_size[0], output_size[1])`` by combining the values contained in blocks.
+
+Non-batched inputs are also supported, in which case the ``input`` has shape ``(C * Product(kernel_size), L)`` and the output has shape ``(C, output_size[0], output_size[1])``.
+
+**Attributes**:
+
+* *strides*
+
+  * **Description**: stride in the sliding blocks in the input spatial dimensions.
+  * **Range of values**: 1D tensor of non-negative integer numbers
+  * **Type**: *T_IND*
+  * **Default value**: [1, 1]
+  * **Required**: *no*
+
+* *dilations*
+
+  * **Description**: controls local stride of the elements.
+  * **Range of values**: 1D tensor of non-negative integer numbers
+  * **Type**: *T_IND*
+  * **Default value**: [1, 1]
+  * **Required**: *no*
+
+* *pads_begin*
+
+  * **Description**: *pads_begin* is a number of zero-value pixels to add to the beginning along each axis. For example, *pads_begin* equal "1,2" means adding 1 pixel to the top of the input and 2 to the left of the input.
+  * **Range of values**: 1D tensor of non-negative integer numbers
+  * **Type**: *T_IND*
+  * **Default value**: [0, 0]
+  * **Required**: *no*
+
+* *pads_end*
+
+  * **Description**: *pads_end* is a number of zero-value pixels to add to the ending along each axis. For example, *pads_end* equal "1,2" means adding 1 pixel to the bottom of the input and 2 to the right of the input.
+  * **Range of values**: 1D tensor of non-negative integer numbers
+  * **Type**: *T_IND*
+  * **Default value**: [0, 0]
+  * **Required**: *no*
+
+**Inputs**
+
+* **1**: *data*
+
+  * **Description**: A batched 3D tensor of type *T* and shape ``(N, C * Product(kernel_size), L)`` or an unbatched 2D tensor of type *T* and shape ``(C * Product(kernel_size), L)``. **Required.**
+  * **Range of values**: 1D tensor of non-negative integer numbers
+  * **Type**: *T*
+
+* **2**: *output_size*
+
+  * **Description**: controls the shape of the spatial dimensions of the output image. **Required.**
+  * **Range of values**: batched 3D or unbatched 2D tensor of non-negative integer numbers
```

Comment:
It doesn't match to `shape_infer` in the other PR.

Reply:
```suggestion
  * **Range of values**: 1D tensor with 3 elements or 2 elements of non-negative integer numbers
```

Reply:
Corrected to `1D tensor of two non-negative integer numbers (height and width)`, following PyTorch's convention.